spec/types/calculation.md

```diff
@@ -192,6 +192,9 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
 * Let `arguments` be the result of [simplifying](#simplifying-a-calculationvalue) each
   of `calc`'s arguments.
 
+* If `calc` is contained within a supports declaration, return a calculation
```

Comment:
"is contained within a supports declaration" is ambiguous... it could apply to calculations that appear within interpolations, or it could even be read to apply to calculations that are in other statements nested inside a `@supports` rule. It's better to refer to specific parser productions, like those in [this proposal](https://github.com/sass/sass/blob/main/accepted/supports-general.md#syntax).

For example, you could say "If `calc` was parsed from an expression within a `SupportsDeclaration`'s `Expression` but outside any interpolation".